### DISCUSSION TITLE
How do I insert previous ans?

### DISCUSSION THREAD

**Discussion by ojsl1** [lcn2/calc#76] [Q&A] [answered]
Hi, I'm new to using calc, the one and probably the only feature I need for basic arithmetic is to be able to quickly insert long answers from previous queries to a new prompt (without having to rely on using variables for every small query). Is this not possible in calc? I wasn't able to discern an answer to this from the man page nor the built-in help pages.

**Comment by lcn2** [comment, marked as answer]
The symbol "." (dot), by itself, is the previous value.

For example:

```calc
; 2+2
	4
; .*5
	20
; .^3
	8000
; ./1000
	8
;
```

See:

```calc
; help expression
```

for more information.

We hope this helps.
